scripts/deploy.ts

```diff
@@ -163,43 +169,31 @@ export async function createGovernorClone(
     proposalThreshold: number,
     quorumNumerator: number,
     salt: BytesLike,
-    gasLimit?: number,
+    gasPrice?: BigNumberish,
+    gasLimit?: BigNumberish,
 ): Promise<GovernorImpl> {
-    const governorFactory = await ethers.getContractAt(`GovernorFactory`, governorFactoryAddr);
-
-    let txReceipt: ContractReceipt;
-    if (gasLimit != undefined && gasLimit != 0) {
```

Comment:
FYI: We don't actually need this check `if (gasLimit != undefined && gasLimit != 0)`, since passing undefined aremuments like we do in:

```TypeScript
                {
                    gasPrice: gasPrice,
                    gasLimit: gasLimit,
                },
```

Will handle if they're undefined and do the default approach. So just need to handle the 0 case, which we do in console.ts.